frontend/packages/citizen-frontend/src/map/UnitDetailsPanel.tsx

```diff
@@ -138,6 +138,17 @@ export default React.memo(function UnitDetailsPanel({
             <Label>{t.map.providerType}</Label>
             <Gap size="xs" />
             <div>{t.common.unit.providerTypes[unit.providerType]}</div>
+            {unit.providerType === 'PRIVATE_SERVICE_VOUCHER' && (
+              <>
+                <Gap size="xs" />
+                <div>
+                  <ExternalLink
+                    text={t.common.unit.providerTypes.PRIVATE_SERVICE_VOUCHER}
+                    href={serviceVoucherLinkHref[lang]}
```

Comment:
Would it make sense to have the links just in i18n? That way they'll also get Tamperefied automatically